recipes/namd/3.0/gh200/environments.yaml

```diff
@@ -0,0 +1,17 @@
+namd-single-node:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@12
+  unify: true
+  specs:
+  - cuda@12
```

Comment:
it's good to be a bit more specific, e.g. `cuda@12.4`.
The `package.py` for `cuda` is provided by the site package repo (not spack's builtin repo), so the version of cuda could change when you reubuild with a new cluster configuration.